invisible_cities/core/fit_functions_test.py

```diff
@@ -225,18 +225,20 @@ def test_profile1D_full_range_x(func, xdata, ydata, xrange):
     xp, yp, ye = func(xdata, ydata)
     assert np.all(fit.in_range(xp, *xrange))
 
+#
```

Comment:
I have just tried
```
@mark.skip(reason="There is something wrong with hypothesis")
```
and it crashes anyway because `FLOAT_ARRAY` is a function that returns an actual value, not a generator.